composer.json

```diff
@@ -26,8 +26,8 @@
 	],
 	"require" : {
 		"php" : ">=5.3.10",
-		"oat-sa/tao-core" : "*",
-		"solarium/solarium": "dev-master"
+		"oat-sa/tao-core" : ">=7.8.0",
```

Comment:
@jbout as the latest released oat-sa/tao-core version as for now is 7.4.10 now it's impossible to install application properly with composer.